portfolio/src/main/webapp/comments.html

```diff
@@ -12,18 +12,19 @@
       integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
       integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+    <script src="authentication.js"></script>
     <script src="comments.js"></script>
   </head>
   <body onload=loadCommentsPage()>
     <div id="content">
       <nav class="navbar navbar-expand-lg navbar-light bg-light">
         <div class="collapse navbar-collapse" id="navbarText">
-          <ul class="navbar-nav mr-auto">
+          <ul class="navbar-nav mr-auto" id="navbar-list">
```

Comment:
You can set the `onload=` attribute of this element to renderAuthentication() (and do the same on index.html), which will avoid having to call renderAuthentication() manually on every page where it is used.

Reply:
The onload attribute can only be applied to these elements:
<body>, <frame>, <frameset>, <iframe>, <img>, <input type="image">, <link>, <script> and <style>
Might just have to keep it as is?

Reply:
Ah, in that case, could you make the onload of the body call two functions, as in Meagan's PR?

Reply:
Formatting for reference: <body onload="function1(); function2();">